1-introduction.md

```diff
@@ -0,0 +1,60 @@
+# Introduction
+
+## What is Plasma?
+
+The [whitepaper](http://plasma.io/plasma.pdf), written by Vitalik Buterin and Joseph Poon, describes Plasma as:
+
+> Plasma is a proposed framework for incentivized and enforced execution of smart contracts which is scalable to a significant amount of state updates per second (potentially billions) enabling the blockchain to be able to represent a significant amount of decentralized financial applications worldwide.
+
+That's a lot to digest. More simply, Plasma is a method of securely scaling a blockchain. It's not a framework like _Ruby on Rails_, _Express.js_, or _UIKit_. I think about it as a proposed [design pattern](https://en.wikipedia.org/wiki/Software_design_pattern) to allow a blockchain to represent a lot more state updates (transactions).
+
+
+## Under Construction
+
+Plasma is currently under heavy research and development by OmiseGO and multiple teams around the world. The white paper specifies many concepts that are yet to be explored for implementation. In the [Specifications](specifications) section, you can see more concrete specifications of Plasma, as well as implementations of those specs.
+
+
+## Structure
+
+In the most general case, Plasma describes the interaction of a tree of blockchains.
+
+[ XXX - add graphic ]
+
+Going from the leaf chains to the root of the tree, each chain periodically submits a summary update of its state to the parent chain. Eventually, the update of all the updates get submitted to a root chain (e.g. Ethereum). Conceptually, this is much like MapReduce. We map the update procedure through the tree and repeatedly reduce the results.
```

Comment:
The white paper references MapReduce, but I'm having a little trouble seeing how Map fits here.  At least right now a tree of Plasma chains collects transactions at its tips and then Reduces them up into its parents and eventually the root chain.  Would you mind explaining this more?

Reply:
I'm looking at it as an implicit Map, as enforced by the protocol. Am I understanding this wrong?

Reply:
Can you explain more about how it's being mapped implicitly?